internal/token/jwt.go

```diff
@@ -0,0 +1,78 @@
+package token
+
+import (
+	"errors"
+	"github.com/benbjohnson/clock"
+	"github.com/golang-jwt/jwt/v5"
+	"kg/procurement/cmd/config"
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type jwtManager struct {
+	cfg   config.Token
+	clock clock.Clock
+}
+
+func (s *jwtManager) GenerateToken(spec ClaimSpec) (string, error) {
+	if s.cfg.Secret == "" {
+		return "", errors.New("secret key is empty")
+	}
```

Comment:
I think the `validate` on the config declaration already handles this

Reply:
This is for double protection i guess